dex/farm-staking-proxy/wasm/src/lib.rs

```diff
@@ -7,7 +7,6 @@
 elrond_wasm_node::wasm_endpoints! {
     farm_staking_proxy
     (
-        init
```

Comment:
no init ?

Reply:
yep... strange, the init is still there, compiled it manually on my machine, but this auto generated code does not include it anymore for some reason, probably the rust framework minor version changed and it just doesnt generate init anymore